frontend/test/metabase-smoketest/user.cy.spec.js

```diff
@@ -260,11 +254,9 @@ describe("smoketest > new_user", () => {
     cy.get(".Icon-close")
       .last()
       .click();
+  });
```

Comment:
If we change the row limit, we should validate that the resulting table only loads 10 rows.